exercises/concept/cater-waiter/.meta/exemplar.py

```diff
@@ -8,11 +11,11 @@
 
 
 def clean_ingredients(dish_name, dish_ingredients):
-    """
+    """Remove duplicates from `dish_ingredients` and return a tuple containing `dish_name` and the cleaned list.
```

Comment:
```suggestion
    """Remove duplicates from `dish_ingredients`.
```

Reply:
My logic here is that the remainder of the docstring describes the return content and type.